packages/tocco-util/src/form/formBuilder.js

```diff
@@ -135,20 +135,33 @@ export default (
   }
 
   const createLayoutComponent = (field, type, key, traverser) => {
-    if (type === layoutTypes.HORIZONTAL_BOX || type === layoutTypes.VERTICAL_BOX) {
-      const alignment = type === layoutTypes.HORIZONTAL_BOX ? 'horizontal' : 'vertical'
-      const children = traverser()
+    let output = traverser()
```

Comment:
I've tried to do no reassignments and name variables better (see d8e66ed21e08f60eb244fefd9ef30bca16d52a12). I am not happy with the result because I could not achieve neither in a reasonable amout of time. May give me a hint.